canvas_modules/common-canvas/src/palette/palette-content-list-item.jsx

```diff
@@ -299,6 +306,8 @@ class PaletteContentListItem extends React.Component {
 				className={mainDivClass}
 				onMouseOver={this.onMouseOver}
 				onMouseLeave={this.onMouseLeave}
+				tabIndex={0}
```

Comment:
It's only a minor thing but can you move this up a couple of lines so that it is not stuck in the middle of all the event handler props.?

Reply:
tabIndex={0} has been moved above event handler props.